server/proxy/expand.go

```diff
@@ -261,7 +291,12 @@ func stripApp(app apps.App, level apps.ExpandLevel) *apps.App {
 	return nil
 }
 
-func getExpandClient(app apps.App, conf config.Config, mm *pluginapi.Client, in Incoming) (mmclient.Client, error) {
+func (p *Proxy) getExpandClient(app apps.App, in Incoming) (mmclient.Client, error) {
+	if p.expandClientOverride != nil {
+		return p.expandClientOverride, nil
+	}
```

Comment:
If a `pluginapi.Client` is passed to `getExpandClient` this workaround is not longer needed. Why does the `Proxy` need a `expandClientOverride`?